ynr/apps/moderation_queue/templates/moderation_queue/includes/lock_suggestions.html

```diff
@@ -1,165 +0,0 @@
-<p>Suggestions are grouped by nomination paper</p>
```

Comment:
is this just dead code that should have been removed in #792 ?

Reply:
Nah, it's an include I've stopped using in this PR https://github.com/DemocracyClub/yournextrepresentative/pull/863/files/79b5e0afd716451d301719f202d827f2cf1d9e71#diff-1175212586b450466bc4c4eec28fd9feL24